### PR TITLE
Added SEO Related settings

### PR DESCRIPTION
**Pull request by githubprabin143**
**Added SEO Related settings**, so user can set document's title and description using seo setting mention on the portfolio.js file.

### CODE REVIEW

File: src/portfolio.js [hunk 1]
```diff
@@ -5,6 +5,18 @@ const settings = {
...
 
+//SEO Related settings
+const seo = {
+  title: "Ashutosh's Portfolio",
+  description:
+    "A passionate individual who always thrive to work on end to end products which develop sustainable and scalable social and technical systems to create impact.",
+  og: {
+    title: "Ashutosh Hathidara Portfolio",
+    type: "website",
+    url: "https://ashutosh1919.github.io/",
```
Comment (ashutosh1919) [inline src/portfolio.js:16]:
```suggestion
    url: "http://ashutoshhathidara.com/",
```

File: public/index.html [hunk 2]
```diff
@@ -5,8 +5,8 @@
     <!-- <link rel="icon" href="%PUBLIC_URL%/favicon1.ico" /> -->
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta
-      name="Ashutosh's Portfolio"
```
Comment (ashutosh1919) [inline public/index.html:8]: Any Reason behind removing this?

**Review by ashutosh1919 — CHANGES_REQUESTED** [review]
I was looking for a PR for SEO. Thanks for raising this one. Hoping to merge it soon. Resolve few things and we are good to merge.
@githubprabin, please push back the package-lock.json file. I accidently deleted it.

File: src/portfolio.js (hunk 1 above)
Comment (githubprabin143) [reply src/portfolio.js:16]: Ok @ashutosh1919 . Thank you :)

File: public/index.html (hunk 2 above)
Comment (githubprabin143) [reply public/index.html:8]: No reason @ashutosh1919 i will put it back as it is.

**Comment by githubprabin143** [comment]
I resolved the things please have a look @ashutosh1919 . Let me know if anything is missing or need to resolve.

**Comment by ashutosh1919** [comment]
@githubprabin143 Thanks for contribution. Keep contributing such stuff on the repository. :)

**Comment by ashutosh1919** [comment]
@all-contributors please add @githubprabin143 for code

**Comment by githubprabin143** [comment]
Thank you so much for adding me as a contributor :) .

**Pull request merged**